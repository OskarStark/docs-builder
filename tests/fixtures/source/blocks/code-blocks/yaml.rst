
.. code-block:: yaml
    # some code
    parameters:
        credit_card_number: 1234_5678_9012_3456
        long_number: 10_000_000_000
        pi: 3.14159_26535_89793
        hex_words: 0x_CAFE_F00D
        canonical: 2001-12-15T02:59:43.1Z
        iso8601: 2001-12-14t21:59:43.10-05:00
        spaced: 2001-12-14 21:59:43.10 -5
        date: 2002-12-14
