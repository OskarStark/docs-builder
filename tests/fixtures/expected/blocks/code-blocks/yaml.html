<div translate="no" data-loc="10" class="notranslate codeblock codeblock-length-md codeblock-yaml">
    <div class="codeblock-scroll">
        <pre class="codeblock-lines">1
2
3
4
5
6
7
8
9
10</pre>
        <pre class="codeblock-code">
            <code>
                <span class="hljs-comment"># some code</span>
                <span class="hljs-attr">parameters:</span>
                <span class="hljs-attr">credit_card_number:</span>
                <span class="hljs-number">1234_5678_9012_3456</span>
                <span class="hljs-attr">long_number:</span>
                <span class="hljs-number">10_000_000_000</span>
                <span class="hljs-attr">pi:</span>
                <span class="hljs-number">3.14159_26535_89793</span>
                <span class="hljs-attr">hex_words:</span>
                <span class="hljs-number">0x_CAFE_F00D</span>
                <span class="hljs-attr">canonical:</span>
                <span class="hljs-number">2001-12-15T02:59:43.1Z</span>
                <span class="hljs-attr">iso8601:</span>
                <span class="hljs-number">2001-12-14t21:59:43.10-05:00</span>
                <span class="hljs-attr">spaced:</span>
                <span class="hljs-number">2001-12-14 21:59:43.10 -5</span>
                <span class="hljs-attr">date:</span>
                <span class="hljs-number">2002-12-14</span>
            </code>
        </pre>
    </div>
</div>
